jenkins/catapult/compilers/msvc-2019.yaml

```diff
@@ -0,0 +1,9 @@
+c: msvc
+cpp: bar
+version: 16
+
+stl:
+  version: baz
+  lib: marker
+
+deps:
```

Comment:
are we using 2019 anywhere?

Reply:
no its not in  the daily builds.  Should I create a weekly job or remove?